ocpkg

```diff
@@ -315,6 +316,9 @@ PACKAGES_EXDEV="
 		libtool \
 		libunistring0 \
 		libxss1 \
+                libxext6 \
+                libsm6 \
+                libxrender1 \
 		make \
```

Comment:
what are these libraries for?

Reply:
You're right we can do without these, they are for the docker version of notebooks.